tests/features/ecas-register.feature

```diff
@@ -9,3 +10,40 @@ Feature: Register through OE Authentication
     # Redirected to the Ecas mockup server.
     Then I should see "Create an account"
     And I should see "This page is part of the EU login mock."
+
+  @cleanup:user
+  Scenario: Login user with auto-register enabled
+    Given I am an anonymous user
+    Given CAS users:
```

Comment:
Normally, if we have many `Given`s, only the 1st one is `Given`, the following should be `And`. Also, I would swap the order because `I am an anonymous user` starts the UI journey while `CAS users` is using API to create the CAS users.